Api/V8/Service/ModuleService.php

```diff
@@ -200,11 +201,25 @@ public function createRecord(CreateModuleParams $params, Request $request)
         $this->setRecordUpdateParams($bean, $attributes);
 
         foreach ($attributes as $property => $value) {
```

Comment:
PSR-2 (Cyclomatic complexity could also potentially be reduced here, not required though as the method still meets the standard): 

```php
        foreach ($attributes as $property => $value) {
            if ($property === 'filecontents') {
                continue;
            }
            if ($property === 'filename') {
                $createFile = true;
            } else {
                $bean->$property = $value;
            }
        }

        if ($createFile) {
            $this->addFileToNote($bean, $attributes);
        }
```